examples/patterns/accordion.html

```diff
@@ -52,14 +52,13 @@
     .querySelector(accordionContainerSelector)
     .addEventListener('click', e => {
       const target = e.target;
-      if (target.classList.contains('p-accordion__tab')) {
```

Comment:
We still need this conditional otherwise the function will run when you click the accordion panel too (which doesn't have an aria-expanded attribute)